core-common/src/main/java/org/zalando/nakadi/domain/HeaderTag.java

```diff
@@ -0,0 +1,25 @@
+package org.zalando.nakadi.domain;
+
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.Optional;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public enum HeaderTag {
+    CONSUMER_SUBSCRIPTION_ID;
+
+    private static final Map<String, HeaderTag> STRING_TO_ENUM = HeaderTag.
+            stream().
```

Comment:
Minor: this can be inlined (the `stream()` method won't be used then)
```suggestion
    private static final Map<String, HeaderTag> STRING_TO_ENUM = Arrays.stream(values()).
```